app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_post_counter(author)
+    author.update(posts_counter: author.posts_counter + 1)
+  end
+
+  validates :title, presence: true, length: { maximum: 25 }
```

Comment:
- [ ] please title maximum must not exceed 250 characters, not 25. please correct that